strapi/strapi
# replace every axiosInstance with useFetchClient or getFetchClient

### What does it do?

## Replace every instance of axiosInstance with the new useFetchClient and getFetchClient

We have already implemented the new constructs useFetchClient and getFetchClient in[ this PR](https://github.com/strapi/strapi/pull/14759)

Now we need to replace every place where we use axiosInstance with the new constructs, test the possible regression errors and still provide the old implementation of the axiosInstance into the helper plugin with a message of the removal in v5

### Why is it needed?

Inside the Strapi project we have a lot of different implementations of the same code to use axios and its axiosInstance.

For example the axiosInstance is defined in:

- *****admin*****
- ***users-permissions***
- ******upload******
- ****i18n****
- *content-type-builder*

### How to test it?

You need to use the new construct inside every axiosInstance call and check the possible regression errors
